src/main/java/roomescape/controller/ReservationController.java

```diff
@@ -0,0 +1,43 @@
+package roomescape.controller;
+
+import jakarta.validation.Valid;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import roomescape.domain.Reservation;
+import roomescape.domain.Reservations;
+import roomescape.dto.request.ReservationRequest;
+import roomescape.dto.response.ReservationResponse;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+
+@Controller
+@RequestMapping("/reservations")
+public class ReservationController {
+
+    private final AtomicLong counter = new AtomicLong(1);
+    private final Reservations reservations = new Reservations(new ArrayList<>());
```

Comment:
```suggestion
    private final Reservations reservations = new Reservations();
```

빈 리스트로 생성하는 경우에는 아무 인자를 전달하지 않고, 내부에서 기본생성자를 정의하여 `new ArrayList<>()` 를 만들어줄 수 있을 것 같습니다.
어떻게 생각하시나요?

Reply:
좋습니다! 더 간단해질 것 같아요!

`Reservations`의 테스트 용이성을 위해 기존에 리스트를 받는 생성자는 유지하는 것이 좋을까요? 현재 테스트에서 사용하지는 않지만, 후에 예약 여러개를 한번에 추가할 때 사용할 수 있을 것 같습니다.